axlearn/common/trainer_config_modifier.py

```diff
@@ -17,7 +18,42 @@
 from axlearn.common.gradient_accumulation import with_minibatch_steps
 from axlearn.common.metrics import MetricAccumulator
 from axlearn.common.trainer import SpmdTrainer
-from axlearn.common.utils import HybridMeshShape, MeshShape
+from axlearn.common.utils import HybridMeshShape, MeshShape, PartitionSpec
+
+
+def find_target_module(
+    module_name: str, cfg: SpmdTrainer.Config
+) -> tuple[ConfigModifier.Config, ConfigModifier.Config, ConfigModifier.Config]:
```

Comment:
Do not return tuples. Return a struct or namedtuple:

https://github.com/apple/axlearn/blob/main/docs/ml_api_style.md#avoid-returning-a-tuple-of-values